blackbox_solver/src/barretenberg/wasm/scalar_mul.rs

```diff
@@ -14,12 +17,46 @@ impl ScalarMul for Barretenberg {
     fn fixed_base(
         &self,
         low: &FieldElement,
-        _high: &FieldElement,
+        high: &FieldElement,
     ) -> Result<(FieldElement, FieldElement), Error> {
         let lhs_ptr: usize = 0;
         let result_ptr: usize = lhs_ptr + FIELD_BYTES;
-        self.transfer_to_heap(&low.to_be_bytes(), lhs_ptr);
 
+        let low_bytes = low.to_be_bytes();
+        let high_bytes = high.to_be_bytes();
```

Comment:
Why not use `try_into_u128` here? This would make readability of this function much nicer.

Reply:
That method is doing a lot of allocations, so I decided to not go that route. Happy to go with `try_into_u128` for now as this method is mostly temporary